### PR TITLE
Support for Visual Studio 2022

### PR DESCRIPTION
**Pull request by tkaczz**
https://github.com/microsoft/RoslynClrHeapAllocationAnalyzer/issues/90

### CODE REVIEW

**Comment by joperezr** [comment]
@mjsabby Since VS 2022 is out now, can somebody please approve and merge this?

File: ClrHeapAllocationsAnalyzer.Vsix/source.extension.vsixmanifest [hunk 1]
```diff
@@ -9,9 +9,24 @@
...
+		  <ProductArchitecture>amd64</ProductArchitecture>
+		</InstallationTarget>
+        <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[15.0,)">
+			<ProductArchitecture>x86</ProductArchitecture>
+		</InstallationTarget>
+		<InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[17.0,18.0)">
+		  <ProductArchitecture>amd64</ProductArchitecture>
+		</InstallationTarget>
+        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[15.0,)">
+			<ProductArchitecture>x86</ProductArchitecture>
```
Comment (mjsabby) [inline ClrHeapAllocationsAnalyzer.Vsix/source.extension.vsixmanifest:25]: Can you fix up the tabs here? And use spaces instead in this entire file?

**Review by mjsabby — CHANGES_REQUESTED** [review]

**Comment by mjsabby** [comment]
@joperezr I'm no longer at Microsoft, perhaps @davidfowl and you can figure out some one in the Roslyn team that can take over this. The changes look fine.

**Comment by davidfowl** [comment]
cc @jahredpar @stephentoub?

**Comment by stephentoub** [comment]
Looks fine to me (tabs aside), but I'm not a vsix expert nor do I have write access.

File: ClrHeapAllocationsAnalyzer.Vsix/source.extension.vsixmanifest (hunk 1 above)
Comment (tkaczz) [reply ClrHeapAllocationsAnalyzer.Vsix/source.extension.vsixmanifest:25]: No problem, just later.

Comment (tkaczz) [reply ClrHeapAllocationsAnalyzer.Vsix/source.extension.vsixmanifest]: @mjsabby done

**Comment by JimBobSquarePants** [comment]
What can Joe Public do here to help get this merged? Would be great to have this up and running again.

**Comment by stephentoub** [comment]
@terrajobst, can you help figure out who has merge rights here?

**Comment by JimBobSquarePants** [comment]
I hate to be a pest but while the PR is merged (_many thanks!_) a Release hasn't been created. Would it be possible to do that please?

**Pull request merged**